src/styles/home.scss

```diff
@@ -35,8 +35,9 @@
   }
   .learn-more {
     background-image: linear-gradient(to bottom, #de4270, #1f2b44);
-    border: 0;
+    border-bottom-width: 0;
```

Comment:
try using border: 0px solid; to be sure that the browser default is overwritten

Reply:
@himankpathak it's still there 

![image](https://user-images.githubusercontent.com/8433031/47264795-a68fa600-d51d-11e8-8061-dba0c857c0e0.png)

Reply:
`border: 0px solid;` was the previous value and it did not fix the issue

![screenshot from 2018-10-21 14-12-04](https://user-images.githubusercontent.com/26011845/47264823-62130300-d53b-11e8-87e5-fb7ee1dc198e.png)

Reply:
@alexdor weird! on local, there is no white line in the browser but in netlify I can see the white line too.